protocol/0056-REWA-rewards_overview.md

```diff
@@ -259,12 +259,34 @@ Recurring transfers can target groups of markets, or all markets for a settlemen
 
-The entire reward account balance is paid out every epoch unless the total value of the metric over all entities is zero, in which case the balance will also be zero anyway (there are no fractional payouts).
+If a `target_notional_volume` is specified in the [recurring transfer](./0057-TRAN-transfers.md#recurring-transfers-to-reward-accounts), the amount to distribute as rewards is first scaled as per the mechanics defined in section [reward scaling](#reward-scaling). The scaled amount is then paid out unless the total value of the metric over all entities is zero.
 
 Rewards are first [distributed amongst entities](#distributing-rewards-amongst-entities) (individuals or teams) and then any rewards distributed to teams are [distributed amongst team members](#distributing-rewards-amongst-team-members).
 
 Any rewards earned by an AMM sub-key should be sent as normal to the relevant vesting account for that sub-key. The party owning the sub-key will be able to withdraw any vested rewards using a regular one-off transfer specifying a `from` key (as per the mechanics detailed [here](./0057-TRAN-transfers.md)), or alternatively leave the reward in the vesting / vested accounts to receive a multiplier on any future rewards (as per the mechanics detailed [here](./0085-RVST-rewards_vesting.md#clarification-for-amm-sub-accounts).
 
+
+### Reward scaling
+
+Before distributing rewards amongst entities, in order to prevent over funding rewards when volume is low, the actual amount transferred and distributed can be scaled based on the target notional volume and the actual notional volume over the reward window (the size of the window specified in the recurring transfer).
+
+$$T_a = T\cdot\min{(1, \frac{\sum_{i}^{n}V_i}{V_t})}$$
+
+Where:
+
+- $T_a$ is the actual amount of funds to transfer and distribute amongst entities.
+- $T$ is the transfer amount specified in the recurring transfer.
+- $V_t$ is the target average notional volume specified in the recurring transfer (represented in asset decimals).
+- ${V_i}$ is the actual notional volume across all markets within scope in epoch $i$ (represented in asset decimals).
+
+Note the following considerations:
+
+- If no target notional volume is specified in the recurring transfer, no scaling takes place. The full transfer amount is always distributed if possible.
+- If the actual notional volume is greater than the target notional volume, the rewards will not be scaled (as the scaling factor will be `1`). The final transfer amount cannot be greater than the transfer amount specified in the recurring transfer.
+- reward scaling is not possible for the following reward metrics.
+  - [Validator ranking](#validator-ranking-metric)
+  - [Market creation](#market-creation-reward-metrics)
+
```

Comment:
what about eligible entities? (specifically because no asset needs to be defined)

Reply:
Good point, will update this PR now the eligible entities spec is merged.

We will definitely need to make it so reward scaling can not be used when no asset is defined (i.e. in these [cases](https://github.com/vegaprotocol/specs/blob/suzukacastle/protocol/0056-REWA-rewards_overview.md#:~:text=the%20cases%20where%3A-,it%20specifies%20no%20metric%20asset%2C%20no%20markets%20within%20the%20market%20scope,.,-If%20however%20a)). Think it should be allowed in the case when an asset is defined (i.e. this [case](https://github.com/vegaprotocol/specs/blob/suzukacastle/protocol/0056-REWA-rewards_overview.md#:~:text=If%20however%20a%20position%20requirement%20is%20specified%2C%20an%20asset%20must%20be%20specified%20also%20and%20then%20parties%20must%20meet%20the%20position%20requirement%20to%20receive%20rewards.)).